src/languages/php.js

```diff
@@ -394,6 +393,42 @@ export default function(hljs) {
     ]
   };
 
+  const FUNCTION_INVOKE = {
+    relevance: 0,
+    match: [
+      /\b/,
+      // to prevent keywords from being confused as the function title
+      regex.concat("(?!fn\\b|function\\b|", normalizeKeywords(KWS).join("\\b|"), "|", normalizeKeywords(BUILT_INS).join("\\b|"), "\\b)"),
+      IDENT_RE,
+      regex.concat(WHITESPACE, "*"),
+      regex.lookahead(/(?=\()/)
+    ],
+    scope: {
+      3: "title.function.invoke",
+    }
+  };
+  FUNCTION_INVOKE.contains = [
+    {
+      relevance: 0,
+      begin: /\(/,
+      end: /\)/,
+      keywords: KEYWORDS,
+      contains: [
+        {
+          scope: 'attr',
+          match: IDENT_RE + regex.lookahead(':') + regex.lookahead(/(?!::)/),
```

Comment:
These may not always return strings, please use `regex.concat`.

Reply:
Done